public/themes/custom/hdbt_subtheme/hdbt_subtheme.theme

```diff
@@ -301,7 +301,7 @@ function hdbt_subtheme_preprocess_field(&$variables) {
     'zh' => t('Chinese', [], ['context' => 'Language of instruction']),
   ];
 
-  foreach ($variables['items'] as $delta => &$item) {
+  foreach ($variables['items'] as &$item) {
```

Comment:
Why is this change done?

Reply:
Delta wasn't used and php linter complained about it so I removed in the name of cleaning.

Reply:
Perfect!